Features/kind nginx script

rafaelstamatie opened the pull request:
The script creates a local Kubernetes cluster using Kind and starts a Pod with the nginx image in the default namespace.

rafaelstamatie committed:
Deploy swarm wordpress comments added

rafaelstamatie committed:
Added basic script to create kind cluster with nginx pod

ionutbalutoiu reviewed (commented):
Can you please drop all the changes to `deploy-swarm-wordpress.sh` ?
They shouldn't be in this pull request.

Regarding `kind-nginix.sh`, it should be named `kind-nginx.sh`.
This is a good start on using Kubernetes with `kind`.

Can you please submit the YAML file corresponding to this command?
```
 kubectl run nginx --image=nginx --port=80
```

In Kubernetes world, YAML files (also called manifests) are used to describe the actual K8s resources to be created, and you can do:
```
kubectl apply -f <yaml_file_path>
```
to create all of them together.

You should stick to submitting YAML files for every K8s work that you do.

Commit:
Add fixed script and restructure it

Commit:
Removed old files from this

rafaelstamatie commented:
> Can you please drop all the changes to `deploy-swarm-wordpress.sh` ? They shouldn't be in this pull request.
> 
> Regarding `kind-nginix.sh`, it should be named `kind-nginx.sh`. This is a good start on using Kubernetes with `kind`.
> 
> Can you please submit the YAML file corresponding to this command?
> 
> ```
>  kubectl run nginx --image=nginx --port=80
> ```
> 
> In Kubernetes world, YAML files (also called manifests) are used to describe the actual K8s resources to be created, and you can do:
> 
> ```
> kubectl apply -f <yaml_file_path>
> ```
> 
> to create all of them together.
> 
> You should stick to submitting YAML files for every K8s work that you do.

>Regarding deploy-swarm-wordpress.sh, I have modified the script and confirmed it’s fully functional. Since I changed the >initial branch, I will drop those changes from this pull request as requested.

>Thank you for the correction about the script name kind-nginx.sh. I have updated it accordingly, submitted the YAML manifest >for the kubectl run command, and verified everything is working properly with kind.
